api/app/serializers/template_serializer.rb

```diff
@@ -27,10 +27,10 @@
 #==============================================================================
 
 class TemplateSerializer < ApplicationSerializer
-  attributes :name, :extension
-  attribute(:synopsis) { object.metadata[:synopsis] }
-  attribute(:description) { object.metadata[:description] }
-  attribute(:version) { object.metadata[:version] }
+  attributes :name
+  attribute(:synposis) { object.metadata['synopsis'] }
```

Comment:
It should be `synopsis` not `synposis`.

Reply:
d49dcfc8dd68474c56365bde9f96279092275daa